make/go.mk

```diff
@@ -39,6 +39,7 @@ clean-metadata:
 generate-metadata: clean-metadata
 	@echo "generating templates metadata for manifests in $(NSTEMPLATES_BASEDIR)" 
 	@$(foreach tmpl,$(wildcard $(NSTEMPLATES_FILES)),$(call git_commit,$(tmpl),$(NSTEMPLATES_BASEDIR),metadata.yaml))
+	@cat $(NSTEMPLATES_BASEDIR)/metadata.yaml
```

Comment:
Do we still need that?

Reply:
Nope, I was debugging an error I saw but I cannot reproduce it anymore so I will just remove it now.

Reply:
Done https://github.com/codeready-toolchain/host-operator/pull/398/commits/8175aa7d049e469f54424f339896ffb252f1ade9